iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonLayer.m

```diff
@@ -11,39 +11,129 @@
 
 @implementation MSMapGeoJsonLayer
 
--(id)init {
-	if (self = [super init])  {
-		self.fillColor = UIColor.blueColor;
-		self.strokeColor = UIColor.blueColor;
-	}
-	return self;
+- (id)init {
+  if (self = [super init]) {
+    self.fillColor = UIColor.blueColor;
+    self.strokeColor = UIColor.blueColor;
+    self.strokeDashed = NO;
+    self.strokeWidth = 1;
+
+    self.polygonsVisible = YES;
+    self.polylinesVisible = YES;
+    self.iconsVisible = YES;
+  }
+  return self;
 }
 
 - (void)setFillColor:(UIColor *)newfillColor {
-	if (_fillColor != newfillColor) {
-		_fillColor = newfillColor;
-		for (MSMapElement *element in [super elements]) {
-			if ([element isKindOfClass:[MSMapPolygon class]]) {
-				MSMapPolygon *polygon = (MSMapPolygon *)element;
-				polygon.fillColor = _fillColor;
-			}
-		}
-	}
+  if (_fillColor != newfillColor) {
```

Comment:
Can you make a separate pull request for just the formatting changes and then make the code changes in this one? It's more difficult to review when code and formatting changes are intermingled.

Reply:
New PR created for clarity.